test/chart/LineChart.spec.tsx

```diff
@@ -1021,7 +1271,7 @@ describe('<LineChart /> - Rendering two line charts with syncId', () => {
 
   /**
    * Tooltip tries to read payload from Redux but tooltip synchronisation is not in redux yet and so this test fails
-   * TODO implement tooltip sync in redux and then enable this test again
+   *  implement tooltip sync in redux and then enable this test again
```

Comment:
```suggestion
   * TODO implement tooltip sync in redux and then enable this test again
```

I didn't mean to delete this